InsiteTeamTask/Controllers/AttendanceController.cs

```diff
@@ -1,11 +1,13 @@
 ﻿using System.Collections.Generic;
 using InsiteTeamTask.Models;
 using InsiteTeamTask.Repositories;
+using Microsoft.AspNetCore.Cors;
 using Microsoft.AspNetCore.Mvc;
 
 namespace InsiteTeamTask.Controllers
 {
-    [Route("api/[controller]")]
+    [Route("api/values")]
```

Comment:
We like RESTful API routes to be descriptive, so using `api/[controller]` here wasn't an issue since it describes the type of object we are going to be manipulating in this controller. I suspect where you actually needed to change values was in your launch settings, so that starting the API would open `https://localhost:5001/api/attendance` in the browser